vars/getReportsLocationParam.groovy

```diff
@@ -0,0 +1,8 @@
+def call(String testReportsUrl) {
+    def possibleReportLocationsFile = "${testReportsUrl}/reports-locations.json"
+    def reportsLocationParam = [] 
+    if (fileExists(possibleReportLocationsFile)) {
+        reportsLocationParam = ['--reports-json-url', possibleReportLocationsFile]
```

Comment:
I think that `fileExists` does not work on HTTP URLs. According to https://jenkins.io/doc/pipeline/steps/workflow-basic-steps/#code-fileexists-code-verify-if-file-exists-in-workspace it only checks if file exists in the current workspace.

Did you test if this works on HTTP URLs?

Reply:
I see that two copies of `reports-locations.json` are created now. One for `CISelfcheck` and one for `WindowsCompute`. So this `fileExists` would not work otherwise.

I think that for now we should provide a correct path to `reports-locations.json` and in the future (hopefully near future) drop that and just implement some recursive searching of xml and html. It it's feasible of course.

Reply:
>some recursive searching of xml and html
nah - we can just
```
unstash Monitoring
unstash xmlReports
sh " python gather-build-stats.py -xmlReportsDir xmlReports/ ... "
```